app/assets/stylesheets/partials/_variable.scss

```diff
@@ -0,0 +1,6 @@
+$red: rgb(255, 0, 0);
```

Comment:
please be consistent in your notation here i would suggest using 000 everywhere or else go just for 0, but dont mix it up please